bin/format-yarn-outdated.js

```diff
@@ -19,6 +20,7 @@ const cli = meow(`
     --version, -v    Prints the package version.
     --format, -f     Output format. One of either markdown, json or mackerel can be used. Default: markdown
     --excludes, -e   Path to YAML file which specify package names to exclude
+    --generate-yaml  Generate YAML file which specify changelog uris for the packages
```

Comment:
The name is confusing. `excludes.yml` or `changelogs.yml`?
I think `--generate-changelog-yaml` is better. Do you have any other suggestions?